fury/actor.py

```diff
@@ -1210,6 +1210,79 @@ def star(
     )
 
 
+def disk(
```

Comment:
maybe we should rename it `flat_disk`

Reply:
@skoudoro in my opinion `disk` is the best name for this. flat makes it confusing since we have other 2d actors without flat.

Reply:
I understand your point and I finally agree.

Confused, with disk which look like  circle. or it could be a plate.   

Do you agree that 3D disk == cylinder ? 

I need to check the doc to see if we explicity say it is a 2D shape. otherwise, it can be confusing

Reply:
Yes, I think so as well a 3D disk is a cylinder. disk is flat generally and a circle is optimally a round line.

Reply:
ok, sorry for that @ManishReddyR, put it back to disk. thanks